### DISCUSSION TITLE
How do I view the MJML preview?

### DISCUSSION THREAD

**Discussion by dbhathena** [timo-reymann/intellij-mjml-support#174] [Q&A] [answered]
I can't seem to figure out how to view the preview in IntelliJ. I'm using IntelliJ 2023.2 Community Edition, and my `.mjml` file types are recognized as MJML templates by the IDE, the plugin is even showing the syntax helping/highlighting, but I cannot seem to view the preview in any way. Thanks for the help!

**Comment by timo-reymann** [comment, marked as answer]
With the community edition you wont get a preview, as the plugin uses Node.js Integration, which is only available in paid tiers.

There is also #19, that targets this but is not implemented as most people using the plugin have licenses anyway
